src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const DISCOUNT_FOR_SHORT_TERM = 20;
+  const DISCOUNT_FOR_LONG_TERM = 50;
+
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+
+  if (days < SHORT_TERM) {
+    return (PRICE_PER_DAY * days);
+  }
+
+  if (days < LONG_TERM) {
+    return (PRICE_PER_DAY * days) - DISCOUNT_FOR_SHORT_TERM;
+  }
+
+  return (PRICE_PER_DAY * days) - DISCOUNT_FOR_LONG_TERM;
```

Comment:
You write the calculation `PRICE_PER_DAY * days` three times - this can be done once, saving the result in a variable and reusing this variable